### ISSUE TITLE
Investigate low accuracy in XGLM (95.5)

### ISSUE THREAD

**Issue by nemanjagrujic** [tenstorrent/pytorch2.0_ttnn#836]
https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/docs/models/XGLM/input_variations.md

**Comment by bfilipovicTT** [comment]
Softmax is the issue here. Without it accuracy is increased to 98%.
It's the same problem as in CLIP model:

https://github.com/tenstorrent/tt-metal/issues/19781
